### ISSUE TITLE
working config

### ISSUE THREAD

**Issue by laci84** [ZhengyaoJiang/PGPortfolio#6]
Im trying reproduce result plotted in User Guide (10^2), but with default config getting much worse results. Which config was used in example?
Thanks!

**Comment by ZhengyaoJiang** [comment]
The example of user guide has been out-dated. It is the performance on cross-validation set in the old version (with bug). The related hyper-parameters are shown in the appendix of the arxiv article.
We will update the result later.
Besides, you should get around 5 in terms of final portfolio value with default config.

Sorry for that.

**Comment by laci84** [comment]
Ok, thanks!

**Comment by Azael05x** [comment]
Hi, 
when you plan updating user guide and working example?

**Comment by ZhengyaoJiang** [comment]
It is now updated. You should get the same result with default config in the user guide.
Please Note that the data range in the default config is also updated now. Since the hyper-parameters in the default config is tuned on `2015/05/01-2017/05/01`(global data range), It is appropriate to do backtest starting after `2017/05/01`.

**Comment by michaf** [comment]
First, thank you for open sourcing your code! I think it's exemplary scientific practice to publish the exact means of how you obtained a scientific result.
However, I still have difficulties replicating your results using the default config. With the current code and 6 train runs I get in train_summary.csv:

```
     test_pv  backtest_test_pv
0  23.945709         26.156628
1  20.855089         32.774247
2  25.420368         23.789632
3  24.402460         23.660887
4  25.969461         18.094304
5  17.867430         16.008977
```

This seems to be way higher than the aforementioned value of about 5. Am I missing something, or could there still be a problem hidden somewhere?

**Comment by ZhengyaoJiang** [comment]
> This seems to be way higher than the aforementioned value of about 5. Am I missing something, or could there still be a problem hidden somewhere?

The time range has been different in the latest update.
Your result seems a only a bit different from the one in the userguide now. I'm not sure about the reason of this difference though. (maybe the difference of package version or platform would matter)

**Comment by nicolospiezia** [comment]
>  Since the hyper-parameters in the default config is tuned on 2015/05/01-2017/05/01(global data range), It is appropriate to do backtest starting after 2017/05/01.

@ZhengyaoJiang , I'm note sure about the date you mentioned. According to the documentation, the backtest is a portion of the global data range, so it should end `2017/05/01`. If it is appropriate to do backtest starting after `2017/05/01`, which should be the `start/end date` and `test_portion` in the net-config file?

> With the current code and 6 train runs I get in train_summary.csv:

@michaf, could you be so kind to post the `net-config.json` file you used the get the aforementioned results?

**Comment by DexHunter** [comment]
> If it is appropriate to do backtest starting after 2017/05/01, which should be the start/end date and test_portion in the net-config file?

@nicolospiezia Hi! I guess you want to do the paper trading, however it is not open sourced yet. You can try to implement by yourself. If not, then you can check [this comment](https://github.com/ZhengyaoJiang/PGPortfolio/issues/51#issuecomment-369088871) I just gave.
